[docs] Format and examples to specify a tenant/namespace for a topic. (#11526)
apache/pulsar: site2/docs/client-libraries-node.md

```JavaScript
const producer = await client.createProducer({

>   topic: 'my-topic', // or 'my-tenant/my-namespace/my-topic' to specify topic's property and namespace 

Reviewer: ```suggestion
  topic: 'my-topic', // or 'my-tenant/my-namespace/my-topic' to specify topic's tenant and namespace 
```

| Parameter | Description | Default |
| :-------- | :---------- | :------ |

> | `topic` | The Pulsar [topic](reference-terminology.md#topic) to which the producer publishes messages. Format: `<topic-name>` or `<property(tenant)>/<namespace>/<topic-name>`. E.g.: `sample/ns1/my-topic` | |

Reviewer 2: ```suggestion
| `topic` | The Pulsar [topic](reference-terminology.md#topic) to which the producer publishes messages. The topic format is `<topic-name>` or `<property(tenant)>/<namespace>/<topic-name>`. For example, `sample/ns1/my-topic` | |
```
